src/libcollections/str.rs

```diff
@@ -1055,8 +1055,8 @@ impl str {
     /// An iterator over substrings of the given string slice, separated by a
     /// pattern, restricted to returning at most `count` items.
     ///
-    /// The last element returned, if any, will contain the remainder of the
```

Comment:
I'm generally against single letter variable names, but considering the method name is `splitn`, I think `n` makes sense. What do you think @angelsl? If you're alright with it, can you rename the parameter and update the instances of `count` in the doc comment?